### DISCUSSION TITLE
Exposure time finder anycubic

### DISCUSSION THREAD

**Discussion by JORGEPOLO** [sn4k3/UVtools#489] [Q&A] [answered]
## System
 - OS: mac mini1
 - Slicer: lychee
 - Printer: anycubic photon x 6k
 - UVtools version: 3.3.2

The Z axis rises in each exposure, despite having the same layer height, which causes printing failure. I use the anycubic RERF file and uncheck the two options at the end of "Multiple exposures". I have tried it by marking it only one, both and it does not work either, the z axis continues to rise. Some help?

**Comment by sn4k3** [comment, marked as answer]
"Multiple exposures" is not compatible with anycubic machines due thier firmware: https://github.com/sn4k3/UVtools/wiki/Printer-compability-with-per-layer-settings-and-advanced-tools

If you want a RERF like test, you must use "Multiple brightness" section instead and not the "Multiple exposures" for anycubic machines

**Reply by h2ogeek** [reply]
@sn4k3 I would like to use the Multiple brightness option to test test different expsure times! How can I do that? I am familiar with "Multiple exposures" but not with "Multiple brightness".

**Reply by sn4k3** [reply]
The Multiple brightness for fake AA will have an inbuilt dialog/calculator which allow you to select how many steps you want to fade. In that mode you can't choose time for each block, instead you choose how many times you want to divide your base time. If you set 4s and you have AA=4 it means you will have 4 decremental steps. The tool helps you to define the right divisor and also the right AA to set. Just remember you can't set intermediate times, they are defined by AA and shown by UVtools. Give it a try, but do a dry test first as I'm not sure it will work with your machine.

Sample (Exposure=4s & AA=8 & Steps=4), which generate 0.5s steps:

![R_E_R_F_layer20](https://user-images.githubusercontent.com/113281/219884706-a054558a-d7d6-4e9a-89bd-d7630e7325aa.png)

**Reply by h2ogeek** [reply]
Thank you @sn4k3 ! I will try and let you know!
